chain/client/src/client.rs

```diff
@@ -1041,9 +1053,21 @@ impl Client {
                 source: StorageDataSource::Db,
                 state_patch: Default::default(),
             };
+            let prev_chunk_transactions_size = match prev_chunk_opt {
+                Some(prev_chunk) => borsh::to_vec(prev_chunk.transactions())
```

Comment:
assuming this serialization will take some time, guard this with the new protocol version?